frontend/README.md

```diff
@@ -73,3 +73,12 @@ In order of operation:
 We follow Create React App's [approach to environment variables](https://create-react-app.dev/docs/adding-custom-environment-variables/#adding-development-environment-variables-in-env). There is a `.env` file **checked in** that contains non-sensitive default variables. You can create a `.env.local` file on your own machine and it should **not** be checked in.
 
 Environment variables must be prefixed with `REACT_APP_`.
+
+## Notes
+
+It might be beneficial to move the frontend into its own repo. List of reasons why:
```

Comment:
**Question (non-blocking):** Make this a `discussion` issue?

I don't have a specific preference as to 1 or 2 repos.  It's come up before but the project's context has changed now that we're using Node on the backend.  By making this an issue devs working on the project can discuss it in one spot, resolve it once consensus is reached, even re-open it if new trade-offs are discovered down the road.

As decisions are made at this point in the project lifespan, it may also be wise to document them.  @coderbyheart has a nifty "Architectural Decision Record" format we could try out once things begin to stabalize!

Reply:
I'm happy to have a discussion on this and agree that we should move that to an issue and perhaps schedule a time to talk about it. fwiw I'm pretty accustomed to working in monorepos and see some benefits that I'm hesitant to let go of in terms of.

Reply:
I will give it a little more thought and move it to an issue if needed 👍